packages/hw-app-btc/src/Btc.js

```diff
@@ -136,7 +136,8 @@ export default class Btc {
 
   getTrustedInput(
     indexLookup: number,
-    transaction: Transaction
+    transaction: Transaction,
+    isDecred?: boolean = false
```

Comment:
Prefer to pass additionals here and look for decred string because we've seen this doesn't scale

Reply:
I agree ! I made requires changes